.gitlab-ci.yml

```diff
@@ -18,7 +18,7 @@ stages:
   extends: .LoadModules
   stage: buildDependencies
   script:
-    - git clone https://github.com/google/benchmark.git -b v1.4.1 &&
+    - git clone https://github.com/google/benchmark.git -b v1.6.0 &&
       cd benchmark &&
       git clone https://github.com/google/googletest.git -b release-1.8.1 &&
```

Comment:
Wait why are we downloading GTest?  (not expecting us to fix here)

Reply:
I wonder if it was introduced before disabling benchmark testing, that needs gtest.